backend/plugin/app/relay/relay_client.go

```diff
@@ -28,21 +28,39 @@ func NewClient() *Client {
 
 // CreatePayload is the message to create external approval.
 type CreatePayload struct {
-	// TODO(d): content TBD.
+	IssueID     string    `json:"issueId"`
+	Creator     string    `json:"creator"`
+	Title       string    `json:"title"`
+	Description string    `json:"description"`
+	Project     string    `json:"project"`
+	Assignee    string    `json:"assignee"`
+	Statement   string    `json:"statement"`
```

Comment:
nit: we may need to provide a rollout plan in the future, but this is good for now.

Rollout plan means the list of databases and a list of statements, and there could be a list of statements applied to databases.

Reply:
Yeap, so I actually not use the issue payload in the demo.